diff --git a/dist/runtime/plugins/supabase.client.js b/dist/runtime/plugins/supabase.client.js
index 38bc98bacea1c7d797c3c20b3387e0785216b3fc..886e8b9a648d13063929a264d8752bc49ebffbf0 100644
--- a/dist/runtime/plugins/supabase.client.js
+++ b/dist/runtime/plugins/supabase.client.js
@@ -36,10 +36,8 @@ export default defineNuxtPlugin({
     provide("supabase", { client });
     const currentSession = useSupabaseSession();
     const currentUser = useSupabaseUser();
-    nuxtApp.hook("page:start", async () => {
-      const { data } = await client.auth.getClaims();
-      currentUser.value = data?.claims ?? null;
-    });
+    const { data } = await client.auth.getClaims();
+    currentUser.value = data?.claims ?? null;
     client.auth.onAuthStateChange((_, session) => {
       if (JSON.stringify(currentSession.value) !== JSON.stringify(session)) {
         currentSession.value = session;
